onsite-group-location.html

```diff
@@ -5,116 +5,77 @@
 ---
 
 <section class="container">
-
   <div class="row push-ends">
-    <div class="col-md-12">
-      <a href="#">&lt; Back to All Onsite Groups</a>
-    </div>
+    <div class="col-md-12"><a href="#">&lt; Back to All Onsite Groups</a></div>
   </div>
   <div class="row push-bottom">
     <div class="col-md-3">
       <img
-        src="https://crds-media.imgix.net/53Hn1ikskgKccwCyWsmoYC/3b5c7501c3e22a7c642b907da144aa34/locations-mason.jpg?auto=format,compress&w=273&h=200&fit=crop&ixlib=imgixjs-3.3.2" />
+        src="{{ page.location.image.url | imgix: site.imgix }}?{{
+          site.imgix_params.placeholder_sixteen_nine
+        }}"
+        sizes="{{ site.image_sizes.one_fourth }}"
+        style="width: 100%"
```

Comment:
Should we make this a global style? Guessing it's going to be reused for other images using the imgix-optimizer.